src/yang/ietf-queue-policy.yang

```diff
@@ -63,7 +68,7 @@ module ietf-queue-policy {
       case traffic-group-name {
         uses diffserv:traffic-group;
         description
-          "traffic group name";
+          "Traffic group name";
```

Comment:
Better to say

"Choice of when a filter parameter is a traffic group name", or something like that.